roles/swift-ring/tasks/main.yml

```diff
@@ -1,6 +1,5 @@
 ---
 - name: drop our ring configuration
-  copy: src={{ swift_ring.ring_definition_file }} owner=root group=root
-        dest=/etc/swift/ring_definition.yml mode=644 backup=yes
-  notify: setup swift rings
-
+  copy: src={{ swift_ring.ring_definition_file }} owner=swiftops group=swiftops
```

Comment:
Does this work, the copy source? We've seen issues in ansible 2.1 where the double template value wasn't working. If it does work, it should be used above every place that refers to the definition file.